sandbox/src/mocks/r4/NHSDigital-OperationOutcome.json

```diff
@@ -0,0 +1,24 @@
+{
+  "resourceType": "OperationOutcome",
+  "meta": {
+    "profile": [
+      "https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome"
+    ],
+    "lastUpdated": "2022-03-01T10:00:00.42Z"
+  },
+  "issue": [
+    {
+      "severity": "error",
+      "code": "forbidden",
+      "diagnostics": "Example diagnostics message.",
+      "details": {
+        "coding": [
+          {
+            "system": "https://fhir.nhs.uk/CodeSystem/http-error-codes",
+            "code": "REC_BAD_REQUEST"
```

Comment:
My comment may have gotten lost, I think we should remove this example as they're more specific now.